theme/image-editor/imageEditor.less

```
@@ -91,13 +95,67 @@
     background-color: #4B7BEC;
     border: 2px solid #4067b3;
     border-bottom: none;
+    display: flex;
 }
 
 .image-editor-gallery-content {
     flex-grow: 1;
     overflow: hidden;
     position: relative;
     height: 100%;
+    flex: 9;
+}
+
+// Tablet + Mobile
+@media only screen and (max-width: @largestTabletScreen) {
+    .image-editor-gallery-content {
+        flex: 4!important;
+    }
+
+    .filter-tag {
+        padding-top: .7rem!important;
+        padding-bottom: .7rem!important;
+    }
+}
+
+
+// Mobile Only
+@media only screen and (max-width: @largestMobileScreen) {
+    .image-editor-gallery-content {
+        flex: 3!important;
+    }
+
+    .gallery-editor-toggle {
+        margin-left: .2rem!important;
```

```suggestion
        margin-left: .25rem!important;
```

nitpickiest of possible nitpicks but the top-margin here is .25 so matching looks ever so slightly better to my eyes, feel free to take or leave :)